metamodel.zenta

```diff
@@ -1533,46 +1526,15 @@ count(zenta:neighbours($p1,$p2,'contains,1')[@xsi:type='Entity']) = 1</documenta
-              </child>
-            <child xsi:type="zenta:DiagramObject" id="17ff90e2-6981-406f-9ebc-16c75b15bfa3" zentaElement="75c88dc9-f8b4-4315-8bf4-7989930c2f55">
-                <bounds x="612" y="325" width="121" height="72"/>
-                <sourceConnection xsi:type="zenta:Connection" id="48cd6d04-2b0d-4f9b-8a2f-6ac2bb6bde66" source="17ff90e2-6981-406f-9ebc-16c75b15bfa3" target="388578c1-0e0e-4725-8b6c-563cf268ebd8" relationship="65d3fd8d-3ff4-4be3-a6ac-11d34616d68f"/>
-              </child>
-            <child xsi:type="zenta:DiagramObject" id="43c5cb60-d1f6-4117-9653-a25512f9c5c6" targetConnections="fcbb39e2-2050-4aac-ab39-ff9ec2976dd3" zentaElement="e6913ab1-73c3-424a-9724-72196f6f3327">
-                <bounds x="227" y="144" width="-1" height="-1"/>
-              </child>
-            <child xsi:type="zenta:DiagramObject" id="b8e14a33-baec-4272-9301-fe09ade8baff" targetConnections="4631972a-7930-4cf5-9641-a823ce41e203 5135ffdd-52bb-45fb-8ea0-0652639cb11b" zentaElement="537f012f-2ae2-476c-804e-d8d835cd7c54">
-                <bounds x="516" y="144" width="-1" height="-1"/>
-              </child>
-            <child xsi:type="zenta:DiagramObject" id="388578c1-0e0e-4725-8b6c-563cf268ebd8" targetConnections="48cd6d04-2b0d-4f9b-8a2f-6ac2bb6bde66 85de2cf7-e4ce-4e6c-8f1d-e4453e23da04" zentaElement="dc030e1c-7c6c-4479-8030-c001aa814e9d">
-                <bounds x="876" y="144" width="-1" height="-1"/>
-              </child>
-            <child xsi:type="zenta:DiagramObject" id="6cec2481-33c0-4929-b00c-fed9c5ab5161" zentaElement="0fbc4f57-24ed-4896-895e-71a1b8179290">
-                <bounds x="876" y="333" height="88"/>
-                <sourceConnection xsi:type="zenta:Connection" id="85de2cf7-e4ce-4e6c-8f1d-e4453e23da04" source="6cec2481-33c0-4929-b00c-fed9c5ab5161" target="388578c1-0e0e-4725-8b6c-563cf268ebd8" relationship="6fc6add6-5074-417d-90c9-f5df3b949fc9"/>
-              </child>
-         </element><element xmlns:html="http://www.w3.org/TR/xhtml1/transitional" xmlns:fn="http://www.w3.org/2005/xpath-functions" xsi:type="zenta:BasicObject" name="Called by at least 1 Service" id="75c88dc9-f8b4-4315-8bf4-7989930c2f55" ancestor="bb3cdf1a-0bc1-4772-8913-d0567ef7ab1f" parent="ac65e2e9-896e-4c2c-a4c9-3ac7cb0c49f9">
-            <documentation>design|
-count(zenta:neighbours($p1,$p2,'operates on,2')[@xsi:type='Service']) &gt; 0</documentation>
-         </element></folder>
+            <element xsi:type="zenta:BasicObject" name="Entity" id="dc030e1c-7c6c-4479-8030-c001aa814e9d" ancestor="41b4d079-dbd5-4df2-91db-6a596661ab0d"/>
+            <element xsi:type="zenta:BasicObject" name="Only CRUD or existing query operations" id="0fbc4f57-24ed-4896-895e-71a1b8179290" ancestor="bb3cdf1a-0bc1-4772-8913-d0567ef7ab1f">
+              <documentation>design|
+count(zenta:neighbours($p1,$p2,'operates on,2')[@xsi:type='Service']) > 0</documentation>
```

Comment:
This is not the correct expression for this contract.

Reply:
Do you mean that the name of the Constraint should be changed to be more expressive, or that the xpath expression should be changed?